pkg/operator/etcdcertsigner/etcdcertsignercontroller.go

```diff
@@ -272,7 +294,7 @@ func (c *EtcdCertSignerController) syncAllMasterCertificates(ctx context.Context
 	// pod controller to watch. A single secret ensures that a cert change
 	// (e.g. node addition or cert rotation) triggers at most one static pod
 	// rollout. If multiple secrets were written, the static pod controller
-	// might initiate rollout before all secrets had been updated.
+	// might initiate rollout before all secrets had been signerUpdated.
```

Comment:
Nit: assuming this wasn't intended.
```suggestion
	// might initiate rollout before all secrets had been updated.
```

Reply:
thanks! will fix this in the follow-up PR, to not waste your lgtm :)